src/lib/core/components/fullScreenApps/fullScreenMap.tsx

```diff
@@ -169,7 +173,9 @@ function FullScreenMap(props: FullScreenComponentProps) {
     xAxisVariable: appState.overlayVariable,
     computationType: 'pass',
     markerType: appState.markerType,
-    //TO DO: checkedLegendItems and maybe dependentAxisLogScale
+    // Endpoint can't currently handle checkedLegendItems
+    // checkedLegendItems: appState.checkedLegendItems,
```

Comment:
I tested uncommenting this line, and things seem to work. Can you elaborate on what isn't working? Thanks!

Reply:
@dmfalke I have tested this by uncommenting the line, and it seems to work fine for me too.

Reply:
Oh, when I tested it, I couldn't get any overlay variable info or even counts on the markers. If it works for y'all though, that's great!